tests/helpers/launch.js

```diff
@@ -2,11 +2,13 @@ import fs from 'fs'
 import path from 'path'
 
 import Launcher from '../../packages/wdio-cli/src/launcher.js'
+import { launchCallback } from '../../packages/wdio-cli/src/run.js'
 
 export default function launch (...args) {
     const launcher = new Launcher(...args)
 
-    return launcher.run().then((isFailing) => {
+    return launcher.run(launchCallback).then(({ err, exitCode }) => {
```

Comment:
the `onFulfilled` handle should not have this err, result payload. We are mixing callback with promise patterns here.

Reply:
can you please help with this one? 
I should not do here `.then(result => function() {})` ?

Reply:
removed callback